mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -656,6 +656,30 @@ static LogicalResult printOperation(CppEmitter &emitter,
   return success();
 }
 
+static LogicalResult
+printOperation(CppEmitter &emitter,
```

Comment:
You are missing a test for this translation

Reply:
Where do those tests live? I grepped around but couldnt find where the others are.

Reply:
The tests are under `llvm-project/mlir/test/Target/Cpp`

Reply:
Added, thanks